iroha-lib/model/generators/QueryGenerator.hpp

```diff
@@ -9,50 +9,50 @@ namespace iroha_lib
+                                                          const uint64_t created_time);
 
 public:
-    std::shared_ptr<iroha::protocol::Query> generateGetAccount(const std::string& accountId,
+    std::shared_ptr<iroha::protocol::Query> generateGetAccount(const std::string& account_id,
                                                                uint64_t counter,
-                                                               const uint64_t createdTime);
-    std::shared_ptr<iroha::protocol::Query> generateGetAccountAssets(const std::string& accountId,
+                                                               const uint64_t created_time);
+    std::shared_ptr<iroha::protocol::Query> generateGetAccountAssets(const std::string& account_id,
                                                                      uint64_t counter,
-                                                                     const uint64_t createdTime);
-    std::shared_ptr<iroha::protocol::Query> generateGetAccountDetail(const std::string& accountId,
+                                                                     const uint64_t created_time);
+    std::shared_ptr<iroha::protocol::Query> generateGetAccountDetail(const std::string& account_id,
                                                                      uint64_t counter,
-                                                                     const uint64_t createdTime);
-    std::shared_ptr<iroha::protocol::Query> generateGetAccountTransactions(const std::string& accountId,
+                                                                     const uint64_t created_time);
+    std::shared_ptr<iroha::protocol::Query> generateGetAccountTransactions(const std::string& account_id,
                                                                            uint64_t counter,
-                                                                           const uint64_t createdTime,
+                                                                           const uint64_t created_time,
                                                                            const std::optional<std::string*>& first_tx_hash,
                                                                            const std::optional<google::protobuf::Timestamp*>& first_tx_time,
                                                                            const std::optional<google::protobuf::Timestamp*>& last_tx_time,
                                                                            const std::optional<uint64_t> first_tx_height,
                                                                            const std::optional<uint64_t> last_tx_height);
-    std::shared_ptr<iroha::protocol::Query> generateGetAccountAssetTransactions(const std::string& accountId,
+    std::shared_ptr<iroha::protocol::Query> generateGetAccountAssetTransactions(const std::string& account_id,
```

Comment:
The query also needs optional params as `generateGetAccountTransactions`

Reply:
Added.